scss/_forms.scss

```diff
@@ -47,7 +47,9 @@ input.input-contrast,
 .input-contrast {
   background-color: #fafafa;
 
-  &:focus { background-color: #fff; }
+  &:focus {
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.